daft/analytics.py

```diff
@@ -10,13 +10,15 @@
 import os
 import platform
 import random
+import socket
 import time
 import urllib.error
 import urllib.request
 from typing import Any, Callable
 
 from daft import context
 
+_ANALYTICS_ACTIVE = True
```

Comment:
Can we have this in the analytics client instead as internal state? The analytics client itself is already a global singleton, and we should try to avoid having too many global variables.

Reply:
Ah okay, didn't know it was a global singleton. Will update in the next commit asap.

Reply:
Yup, the `_ANALYTICS_CLIENT` below this one is the global client singleton

Reply:
Updated.